### ISSUE TITLE
Cannot get the result like paper when not using pretrained provided

### ISSUE THREAD

**Issue by Duyih25** [mdswyz/IMDer#1]
When I try to reproduce without using your pretrained, the result is very poor. How to produce a powerful pretrained as you provided? Please help me, I want to use this for my work at university. Thank you a lot

**Comment by mdswyz** [comment]
if you want to train a pretrained model, you need to use a small learning rate such as 0.0001 that could finetune the bert embedding well.
